modules/packages/manifests/supervisor.pp

```diff
@@ -0,0 +1,18 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class packages::supervisor {
```

Comment:
If we can get supervisor+python from s3, we may no longer have a homebrew dep, and we won't be blocked from deploying to beta/esr68.

Reply:
The awkwardness here is setting up supervisord to run. As mentioned in the channel, this either means puppet runs regularly (which I don't think it will) to ensure it's still running, or setting up launchd ourselves, which homebrew is currently doing for us. It's a step backwards to have to roll all this ourselves. 
It also reopens the question of whether we just use launchd directly for the scriptworker.